sdk/onlineexperimentation/azure-analytics-onlineexperimentation/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+directory: specification/onlineexperimentation/Azure.Analytics.OnlineExperimentation
+commit: 7b94be99f4c65d06d2b980843ac54234becbbdda
```

Comment:
notice, make sure the commit is on specs repo main branch

Reply:
I wanted to keep the commit hash synchronized across all the SDK repos. There haven't been any changes to these specs since that commit, but I can move it forward if this is critical.

Reply:
The requirement is the commit is on "main" branch. The date of the commit is up-to-you. You can have the commit that was 1 year ago here, as long as it's on "main".

Ran a check and seems it's not.